feature-libs/cart/wish-list/core/store/actions/wish-list.action.ts

```diff
@@ -79,13 +89,28 @@ interface LoadWishListFailPayload {
    * temporary cart used to track loading/error state or to normal wish list entity.
    */
   cartId: string;
-  error: ErrorActionType;
+  error: ActionErrorProperty;
```

Comment:
[NOTE]

I've left the interface to keep benefiting from its useful existing JSDocs